hypervisor/include/arch/x86/mmu.h

```diff
@@ -221,8 +221,8 @@ void destroy_ept(struct acrn_vm *vm);
  * @param[in] vm the pointer that points to VM data structure
  * @param[in] gpa the specified guest-physical address
  *
- * @return INVALID_HPA - the HPA of parameter gpa is unmapping
- * @return hpa - the HPA of parameter gpa is hpa
+ * @retval INVALID_HPA - the HPA of parameter gpa is unmapping
+ * @return the HPA of parameter gpa is hpa
```

Comment:
Combining `@return` and `@retval` creates two sections in the documentation. It may be OK to have both but I think we should still have one additional `@retval` line that reads something like:
`@retval hpa the host physical address mapping to the \p gpa`

Reply:
Thanks for pointing this out! The `@return` seems redundant after adding the additional `@retval`. I added the `@retval` and dropped the `@return`. Does that look good to you?